app/processing/process-payment-request.js

```diff
@@ -18,6 +20,17 @@ const processPaymentRequest = async (scheduledPaymentRequest) => {
 
   const paymentRequests = await transformPaymentRequest(paymentRequest)
   const { deltaPaymentRequest, completedPaymentRequests } = paymentRequests
+
+  // if FRN is closed (SFI only), remove AR
+  if (paymentRequest.schemeId === SFI && config.handleSFIClosures) {

```

Comment:
Could move this logic to the `getClosedFRN` module to keep this one cleaner.  It all helps to return a boolean of whether we want the close behaviour to run or not.

So that module does the check, if it's not SFI or the toggle is disabled, then just return false.

Reply:
logic moved out into the separate function, returning true or false which then dictates if the filter on completedPaymentRequests is carried out